docs/src/model_providers_and_supported_functions.md

```diff
@@ -0,0 +1,30 @@
+```@meta
+CurrentModule = PromptingTools
+```
+
+# Model Providers and Supported Functions
+
+PromptingTools.jl routes AI calls through the use of subtypes of AbstractPromptSchema, which determine how data is formatted and where it is sent. (For example, OpenAI models have the corresponding subtype AbstractOpenAISchema, having the corresponding schemas - OpenAISchema, CustomOpenAISchema, etc.) This ensures that the data is correctly formatted for the specific AI model provider. 
+
+Below is an overview of the model providers supported by PromptingTools.jl, along with the corresponding schema information.
+
+| Abstract Schema         | Schema                    | Model Provider                         | aigenerate | aiembed | aiclassify | aiextract | aiscan | aiimage |
```

Comment:
Why does aiimage have only one tick vs aiscan has many? Aren't they the same logic / dispatch?

Reply:
Yes, I believe that's correct (it's fuzzy because of the model vs provider vs schema difference, but it's a pretty good summary).